api/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM ubuntu:latest
-ENV SERVER_HOST=0.0.0.0
-RUN apt-get update && apt-get install -y \
-  python-pip \
-  python-dev \
-  build-essential
-COPY . /app
-WORKDIR /app
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
-ENTRYPOINT ["python"]
-CMD ["views.py"]
+FROM httpd:2.4.23
+ENV http_proxy=http://proxy.autozone.com:8080/ SERVER_HOST=0.0.0.0
+WORKDIR /opt
+RUN apt-get update && apt-get install --no-install-recommends -y unzip curl python-pip python-dev build-essential && \
+      curl -k https://infosec.autozone.com/files/openam/Apache_v24_Linux_64bit_4.1.0.zip > /var/tmp/agent.zip && \
+      unzip /var/tmp/agent.zip && rm /var/tmp/agent.zip && \
+      echo "{OpenAM Password}" > /var/tmp/pw.txt  && chmod 0400 /var/tmp/pw.txt && \
```

Comment:
Ok, that sounds good. I went ahead and pulled the OpenAM config and setup related stuff from the dockerfile and added a volume for the pw.